### DISCUSSION TITLE
Arduino controller

### DISCUSSION THREAD

**Discussion by kittystore51** [ExpressLRS/ExpressLRS#2223] [Q&A]
Hi,
I'm trying to build a controller with a DIY expressLRS Tx module (which I already have) and would like to control it with arduino or another microcontroller. I would like to ask you if this is possible and if so if you have any documentation/link to share with me. 
Thanks :)

p.s.
I'm new to these things so forgive me if I have said some stupid things.

**Comment by Ale-maker325** [comment]
https://github.com/kkbin505/Arduino-Transmitter-for-ELRS

**Reply by fajarshod** [reply]
Hi
i have remote with TX ranger micro and i have ExpressLRS as RX. but I don't know the code and links that point to it. Can you help me?

I'm sorry because my English words are very bad

**Comment by Ale-maker325** [comment]
https://github.com/danxdz/simpleTx_esp32
